### PR TITLE
fix: update image.go by adding descriptions and examples

### PR DESCRIPTION
**Pull request by Starnop**
**1.Describe what this PR did**
This PR update image.go to add cli long description and examples.
**2.Does this pull request fix one issue?** 

NONE
**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: cli/image.go [hunk 1]
```diff
@@ -77,7 +78,9 @@ func (i *ImageCommand) runImages(args []string) error {
 }
 
 // imagesExample shows examples in images command, and is used in auto-generated cli docs.
-// TODO: add example
 func imagesExample() string {
-	return ""
+	return `$ pouch images ls
+IMAGE ID             IMAGE NAME                                               SIZE
+bbc3a0323522         docker.io/library/busybox:latest                         2.64 KB
+b81f317384d7         docker.io/library/nginx:latest                           1.98 KB`
```
Comment (allencloud) [inline cli/image.go]: It is really strange that both two images above are less than 3KB?
Is it possible? @Ace-Tang @skoo87 
Or something wrong?

Comment (Starnop) [reply cli/image.go]: I will check it. .Just a moment,please

Comment (Starnop) [reply cli/image.go]: Maybe something is wrong, you would like to see this
[root@localhost cli]# pouch pull docker.io/library/redis:alpine
docker.io/library/redis:alpine:                                                   resolved       |++++++++++++++++++++++++++++++++++++++| 
index-sha256:0153c5db97e5852b25c5c4715cf88cff9d9dfd8fef5c4c81df8b4dd1a984abc7:    done           |++++++++++++++++++++++++++++++++++++++| 
manifest-sha256:7a00cd0e9dbdd56d6f2bac59f52b83bfa8e4e4bfdda4459439dd56615efa4f62: done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:ed931789deac842ebed5fa55c722548cf7384a6ed8ec1daf5a7d19b61dbf7ccd:    done           |++++++++++++++++++++++++++++++++++++++| 
config-sha256:ed8544cc83deb9b120eec364cae7878483c496bd64948fde7815e65a57d04d3a:   done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:1160f4abea84cbe2f316db6306839d2704f09a04af763ee493dd92cb066c0865:    done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:a8c53d69ca3aed47d1db25335f3b3c89f755dc18f177ae7a70c4c8f70f8edcbd:    done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:2dbffccae30ef7a74b938eaa2aaed4b067a15e62d0dd059a063bdf272defae8b:    done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:0a00d98b862d89b5296d56266387a533816a1490efc3aa3d6ae5ecc64eed0a1d:    done           |++++++++++++++++++++++++++++++++++++++| 
layer-sha256:6162f8e8bd54d523376aa6c0f9f01b85472a936837e945d59ce6befb07240ebc:    done           |++++++++++++++++++++++++++++++++++++++| 
elapsed: 207.4s                                                                   total:  9.6 Mi (47.6 KiB/s)                                      
[root@localhost cli]# pouch images
IMAGE ID             IMAGE NAME                                               SIZE
bbc3a0323522         docker.io/library/busybox:latest                         2.64 KB
b81f317384d7         docker.io/library/nginx:latest                           1.98 KB
0153c5db97e5         docker.io/library/redis:alpine                           1.99 KB
The result is not matched the total size(9.6 Mi).

Comment (allencloud) [reply cli/image.go]: Yes,  that is true. So I hope @Ace-Tang @skoo87 to come for a look.

Comment (skoo87) [reply cli/image.go]: Yes, the size is error.  Let me to fix it with a new pr. @allencloud

Comment (allencloud) [reply cli/image.go]: Thanks for the work.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**